app/Actions/Album/Archive.php

```diff
@@ -196,7 +206,14 @@ private function compressAlbum(AbstractAlbum $album, array &$usedDirNames, ?stri
 				} catch (InfoException) {
 					// Silently do nothing, if `set_time_limit` is denied.
 				}
-				$zip->addFileFromStream($fileName, $file->read());
+				$zipFileOption = new ZipFileOption();
+				$zipFileOption->setMethod($this->deflateLevel === -1 ? ZipMethod::STORE() : ZipMethod::DEFLATE());
+				$zipFileOption->setDeflateLevel($this->deflateLevel);
+				$zipFileOption->setComment($photo->title);
+				if ($photo->taken_at !== null) {
+					$zipFileOption->setTime($photo->taken_at);
+				}
```

Comment:
I think I understand the purpose of `setTime` but what does the `setComment` do?